lib/pluginmanager/util.rb

```diff
@@ -17,6 +21,7 @@ def self.logstash_plugin?(plugin, version = nil)
       end
     else
       dep = Gem::Dependency.new(plugin, version || Gem::Requirement.default)
+      Gem.sources = Gem::SourceList.from(options[:rubygems_source]) if options[:rubygems_source]
```

Comment:
Do we really need to do this here?
We are already settings up the environment in the `bundler.rb` file? Is the environment correctly configured at that point?